pkg/skuba/actions/node/remove/remove.go

```diff
@@ -39,7 +40,22 @@ func Remove(target string, drainTimeout time.Duration) error {
 
 	node, err := client.CoreV1().Nodes().Get(target, metav1.GetOptions{})
 	if err != nil {
-		return errors.Wrapf(err, "[remove-node] could not get node %s", target)
+		return errors.Wrapf(err, "could not get node %s", target)
```

Comment:
Keep [remove-node] , this is the current log styling in skuba. Please also fix others.